api/desecapi/models.py

```diff
@@ -179,7 +178,7 @@ class Token(ExportModelOperationsMixin('Token'), rest_framework.authtoken.models
     plain = None
 
     def generate_key(self):
-        self.plain = urlsafe_b64encode(urandom(21)).decode()
+        self.plain = urlsafe_b64encode(secrets.token_bytes(21)).decode()
```

Comment:
If you use `secrets.token_urlsafe()`, you can save the `urlsafe_b64encode()` and the `decode()` call

Reply:
done